### PR TITLE
feature: Add docker image option to run coverage reporter

### PR DESCRIPTION
**Pull request by josemiguelmelo**

### CODE REVIEW

**Review by carlospinho04 — APPROVED** [review]

**Review by carlospinho04 — DISMISSED** [review]

File: docs/alternative-ways-of-running-coverage-reporter.md [hunk 1]
```diff
@@ -101,6 +101,14 @@ If you are having any issues with your installation, you can also build the cove
     <path>/java-project$ java -jar ../codacy-coverage-reporter/target/codacy-coverage-reporter-assembly-<version>.jar report
     ```
 
+## Docker
+
+You can use Docker to run our codacy-coverage-reporter:
```
Comment (pauloribeiro-codacy) [inline docs/alternative-ways-of-running-coverage-reporter.md:106]: Makes the name of the tool more consistent across the documentation:

```suggestion
You can use Docker to run Codacy Coverage Reporter:
```

File: docs/alternative-ways-of-running-coverage-reporter.md [hunk 2]
```diff
@@ -101,6 +101,14 @@ If you are having any issues with your installation, you can also build the cove
     <path>/java-project$ java -jar ../codacy-coverage-reporter/target/codacy-coverage-reporter-assembly-<version>.jar report
     ```
 
+## Docker
```
Comment (pauloribeiro-codacy) [inline docs/alternative-ways-of-running-coverage-reporter.md:104]: This is such a convenient way of running the Codacy Coverage Reporter, that this should probably be the first section on the page, right after "The sections below provide details on alternative ways to run or install Codacy Coverage Reporter". What do you think? :nerd_face:

**Review by pauloribeiro-codacy — COMMENTED** [review]
Exciting stuff! :raised_hands:

File: docs/alternative-ways-of-running-coverage-reporter.md (hunk 2 above)
Comment (josemiguelmelo) [reply docs/alternative-ways-of-running-coverage-reporter.md]: You're totally right 👍 
Moved it to the top

**Review by pauloribeiro-codacy — APPROVED** [review]

**Pull request merged**